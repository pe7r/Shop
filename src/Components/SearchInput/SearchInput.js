import React, { Component } from 'react'
import './SearchInput.scss'
export default class SearchInput extends Component {
    render() {
        return (
            <input className="search__input" placeholder="Search"></input>
        )
    }
}
