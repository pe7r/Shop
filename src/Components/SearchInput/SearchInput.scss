.search__input {
    border: none;
    height: 40px;
    margin-left: 20px;
    font-size: 18px;
}