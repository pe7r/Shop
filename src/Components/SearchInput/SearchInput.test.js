import React from 'react';
import SearchInput from './SearchInput'
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';



describe('SearchInput', () => {
    describe('props works correctly', () => {
        it('theme is an empty string', () => {
            const output = shallow(
                <SearchInput placeholder="Search" theme=''/>
            )
            expect(shallowToJson(output)).toMatchSnapshot()
        })
    })
})