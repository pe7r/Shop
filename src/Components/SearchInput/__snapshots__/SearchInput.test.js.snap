// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput props works correctly theme is an empty string 1`] = `
<input
  className="search__input "
  placeholder="Search"
/>
`;
